### ISSUE TITLE
feat: add a github project link button in MenuBar

### ISSUE THREAD

**Issue by Muhammed-Rahif** [Muhammed-Rahif/Notepad#74]
- Add github project link button in MenuBar, on dropdown 'help'.

**Add link button in this dropdown, at bottom:-**
![Screen Shot 2022-01-03 at 22 20 23](https://user-images.githubusercontent.com/73386156/147958193-98b93049-73f3-4184-b236-4d86380fe1c2.png)

- Should add a logo like the above buttons. Icons can be chosen from `react-icons` librery, use a matching comfortable logo from the librery. Select [logo from here.](https://react-icons.github.io/react-icons/search?q=github)

**Comment by vaishnav-sh** [comment]
Hey @Muhammed-Rahif, Great project! I would love to contribute to this issue. However, I don't have any prior experience with React and TypeScript. Let me know if this issue would be fit for me.

**Comment by Muhammed-Rahif** [comment]
@vaishnav-sh Yeah, you can resolve this issue if you know js.

**Comment by Muhammed-Rahif** [comment]
#### To get this issue bit more clear and simple, I mentioning the steps you need complete on this :- 

1. Create a `openLink` function in `src/helpers/index.ts` to open links.
```ts
function openLink(link: string | URL, options?: { newTab?: boolean }) {
  window.open(link, options?.newTab ? "_blank" : "_top");
}
```
2. Create a button for github in MenuBar, and its `onClick` :- 
```ts
onClick={() =>
       openLink(new URL("https://www.google.com"), { newTab: true })
}
```

That's it!

**Comment by vaishnav-sh** [comment]
> @vaishnav-sh Yeah, you can resolve this issue if you know js. May I assign this issue to you ?

Yeah, please can assign me. Thanks!

**Comment by khushbooag4** [comment]
@vaishnav-sh  Are you still working on this issue. If not I want to work on this issue

**Comment by vaishnav-sh** [comment]
@khushbooag4 Yes please! I'm facing some difficulties with my desktop. @Muhammed-Rahif please assign her for this issue. Thank you for the opportunity!

**Comment by khushbooag4** [comment]
![image](https://user-images.githubusercontent.com/88361750/148676766-5dacce20-e22c-4b9a-b116-0b38bd0e1bb5.png)
It is looking fine

**Comment by Muhammed-Rahif** [comment]
@khushbooag4 :+1:

**Comment by Muhammed-Rahif** [comment]
**Resolved on #104.**
